Implemented new docs structure (#53)
dappforce/subsocial-docs: docs/develop/how-to-guides/comments/GettingComments.md

---
id: fetch-comments
title: Fetch Comments
displayed_sidebar: developSidebar
---

> **This section covers how to create and update comments on Subsocial blockchain.**

Reviewer: ```suggestion
**This section covers how to fetch comments on the Subsocial blockchain.**
```
